### PR TITLE
Added support for django 1.11

### PR DESCRIPTION
**Pull request by ahsan-ul-haq**

### CODE REVIEW

**Comment by ahsan-ul-haq** [comment]
@clintonb I made some small changes.

File: tox.ini [hunk 1]
```diff
@@ -15,12 +15,16 @@ deps =
     15: Django >= 1.5, < 1.6
     16: Django >= 1.6, < 1.7
     17: Django >= 1.7, < 1.8
+    14,15,16,17: South==0.8.4
     18: Django >= 1.8, < 1.9
     19: Django >= 1.9, < 1.10
+    110: Django>=1.10,<1.11
+    111: Django>=1.11,<2.0
```
Comment (schenedx) [inline tox.ini]: Why not do `111: Django>=1.11,<1.12`?

**Comment by feanil** [comment]
Any way we could take the original source commit and then make your changes as a second commit @ahsan-ul-haq ?  This will make porting back to them a lot easier in the future?

File: tox.ini (hunk 1 above)
Comment (clintonb) [inline tox.ini]: There is no Django 1.12. Next version is 2.0.

**Comment by ahsan-ul-haq** [comment]
@feanil I could not find a way to do so. Could you guide me if there is one?

**Comment by schenedx** [comment]
@ahsan-ul-haq Basically, you mentioned that this comment had your updates compared to the original PR right?
Can you port in the original PR in the first comment. Then, on a second commit, make your own edits?
This way, we are able to distinguish what is the code you wrote, what is the code the original commit had. It's easier to code review this way.

**Comment by clintonb** [comment]
Replaced by #2

**Pull request closed without merging**